test/account/AccountERC7913.test.js

```diff
@@ -15,15 +15,39 @@ const signerECDSA = ethers.Wallet.createRandom();
 const signerP256 = new NonNativeSigner(P256SigningKey.random());
 const signerRSA = new NonNativeSigner(RSASHA256SigningKey.random());
 
+// Constants for ZKEmail
+const accountSalt = '0x046582bce36cdd0a8953b9d40b8f20d58302bacf3bcecffeb6741c98a52725e2'; // keccak256("test@example.com")
+const selector = '12345';
+const domainName = 'gmail.com';
+const publicKeyHash = '0x0ea9c777dc7110e5a9e89b13f0cfc540e3845ba120b2b6dc24024d61488d4788';
+const emailNullifier = '0x00a83fce3d4b1c9ef0f600644c1ecc6c8115b57b1596e0e3295e2c5105fbfd8a';
+const templateId = ethers.solidityPackedKeccak256(['string', 'uint256'], ['TEST', 0n]);
+
 // Minimal fixture common to the different signer verifiers
 async function fixture() {
   // EOAs and environment
-  const [beneficiary, other] = await ethers.getSigners();
+  const [admin, beneficiary, other] = await ethers.getSigners();
   const target = await ethers.deployContract('CallReceiverMockExtended');
 
+  // DKIM Registry for ZKEmail
+  const dkim = await ethers.deployContract('ECDSAOwnedDKIMRegistry');
```

Comment:
I missed this change from the last PR regarding the use of `ECDSAOwnedDKIMRegistry`.

We should actually use `UserOverrideableDKIMRegistry` which comes from `@zk-email/contracts`.

We soft deprecated `ECDSAOwnedDKIMRegistry` in favour of `UserOverrideableDKIMRegistry` as the latter is overridable and thus provides an escape hatch to users. We should make a stronger note of this in our repo/docs

Since `UserOverrideableDKIMRegistry` is the contract we would use, would it be possible to update all references of `ECDSAOwnedDKIMRegistry` to the that instead?

Example [import](https://github.com/zkemail/email-tx-builder/blob/b49ed9fbc0e87ecaf6dcdb05c4dc0118727d3593/packages/contracts/script/BaseDeployScript.sol#L8)

Reply:
I tried using the `UserOverrideableDKIMRegistry` but it has an initializer and the constructor is already set. I think using ECDSAOwnedDKIMRegistry in the tests is fine. Note that the actual ZKEmailUtils and SignerZKEmail only require the `IDKIMRegistry` interface, so both should be compatible.

Reply:
> but it has an initializer and the constructor is already set

I don't understand this point, can you clarify?

> It's something we can also emphasize more in our contracts. For example, giving concrete instructions to use the UserOverrideableDKIMRegistry wherever the IDKIMRegistry is declared, wdyt?

The main point I would make is that we should not use `ECDSAOwnedDKIMRegistry` in the production contracts and examples. For tests I am not as opinionated about so happy to go with your call, as long as there are explicit docs that state `UserOverrideableDKIMRegistry` is the registry used in production

Reply:
> I don't understand this point, can you clarify?

Yes, so the `UserOverrideableDKIMRegistry` requires initialization and uses `onlyInitializing` through `__Ownable_init`, which complicates the tests setup. I think I should either be able to write true to the `_initializing` bool in `Initializable` (the other alternative is a top-level call). Creating a proxy has the same issue, we could work around it if the `UserOverrideableDKIMRegistry` was abstract so we can initialize during construction.

> The main point I would make is that we should not use ECDSAOwnedDKIMRegistry in the production contracts and examples. 

Yeah, that makes sense. I'm happy to note it in the documentation in a follow up PR :smile